core/domain/skill_services.py

```diff
@@ -232,156 +232,161 @@ def get_all_skill_summaries():
     return skill_summaries
 
 
-def get_skill_summaries_in_batches(
-        num_queries_to_fetch, cursor=None, sort_by=None):
-    """Returns the summaries of all skills present in the datastore.
+def _get_skill_summaries_in_batches(
+        num_items_to_fetch, cursor, sort_by):
```

Comment:
Should the input cursor also be urlsafe? If so please name it accordingly.

Reply:
Done.